libs/community/tests/integration_tests/chat_models/test_naver.py

```diff
@@ -0,0 +1,62 @@
+"""Test ChatNaver chat model."""
+
+from langchain_community.chat_models import ChatClovaX
+
+
+def test_stream() -> None:
+    """Test streaming tokens from ChatClovaX."""
+    llm = ChatClovaX()
+
+    for token in llm.stream("I'm Clova"):
+        assert isinstance(token.content, str)
```

Comment:
i would recommend adding more checks here, for example asserting that the streamed chunks are `AIMessageChunk` objects, etc

Reply:
(similar for other tests)